pyls/rpc_manager.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2017 Palantir Technologies, Inc.
+import logging
+from uuid import uuid4
```

Comment:
I don't like importing functions from modules. I find it makes reading the code much harder (although clearly not in this case).

Reply:
i think its fine in this case